README.md

```diff
@@ -19,12 +19,14 @@
 </div>
 
 ## What is TotalCross?
-
-TotalCross is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
+TotalCross is tools to make easy and fast GUI creation:
+ 
+* KnowCode:  A computer vision project to help developers to build their screen faster based on image made by design using a library to allow run android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;
```

Comment:
Please, change it to:
_* KnowCode:  A computer vision project to help developers build user interfaces in less time. A library that converts a design image to a file that runs android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;_